backends-velox/src/main/scala/org/apache/spark/sql/execution/VeloxColumnarWriteFilesExec.scala

```diff
@@ -119,6 +122,8 @@ class VeloxColumnarWriteFilesRDD(
       val targetFileName = fileWriteInfo.targetFileName
       val outputPath = description.path
 
+      fileNames += targetFileName
```

Comment:
it seems `writePath` does not contain partition partitionFragment ?  we should collect  `partitionFragment + "/" + targetFileName`